src/main/java/com/kammatysiak/medicalclinic/validator/PatientValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.kammatysiak.medicalclinic.validator;
+
+import com.kammatysiak.medicalclinic.exceptions.PatientPasswordValidationException;
+import com.kammatysiak.medicalclinic.model.dto.PasswordClassDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientDTO;
+import com.kammatysiak.medicalclinic.model.entity.Patient;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class PatientValidator {
```

Comment:
1. nie chcemy zeby taka klasa byla rozszerzana -> powinnismy zrobic ze to jest public final class
2. nie chcemy aby mozliwe bylo utworzenie w ogole instancji takiej klasy -> nie ma to sensu bo sa same metody statyczne -> w tym celu powinnismy zdefiniowac prywatny konstruktor bezargumentowy mozemy to zrobic przy uzyciu lomoboka nad klasa:
@NoArgsConstructor(access = AccessLevel.PRIVATE)